packages/agent/src/certificate.test.ts

```diff
@@ -1,13 +1,14 @@
 /**
- * Need this to setup the proper ArrayBuffer type (otherwise in Jest ArrayBuffer isn't
+ * Need this to setup the proper ArrayBuffer type (otherwise in jest ArrayBuffer isn't
  * an instance of ArrayBuffer).
- * @jest-environment node
+ * @jest-enjestronment node
```

Comment:
is this a stray search/replace, or is "enjestronment" a thing?

Reply:
```suggestion
 * @jest-environment node
```

Yes, it happened accidentally while I was testing to see if the `vitest` library behaved differently from Jest